src/constants/wallets.ts

```diff
@@ -195,3 +195,4 @@ export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
 export type SUPPORTED_WALLET = keyof typeof SUPPORTED_WALLETS
 
 export const WALLETLINK_LOCALSTORAGE_NAME = '-walletlink:https://www.walletlink.org:Addresses'
+export const LS_LAST_WALLETKEY = 'last-wallet-key'
```

Comment:
`LOCAL_STORAGE_LAST_WALLET_KEY` is better

Reply:
fixed